util/gas.ts

```diff
@@ -85,7 +85,11 @@ function sstoreCost(common: Common, inputs: any): BN {
     }
   } else if (common.gteHardfork('istanbul')) {
     if (inputs.newValue === inputs.currentValue) {
-      return new BN(common.param('gasPrices', 'sstoreNoopGasEIP2200'))
+      if (common.gteHardfork('berlin') && inputs.cold !== '1') {
+        return new BN(common.param('gasPrices', 'warmstorageread'))
+      } else {
+        return new BN(common.param('gasPrices', 'sstoreNoopGasEIP2200'))
+      }
```

Comment:
nit: unnecessary else (similarly below)

Reply:
I don't see it? "sstoreNoopGasEIP2200" is not anywhere else, and if I don't return here it will never reach a "return" statement either.

Reply:
Or do you mean I can just remove the "else" keyword? That's a style preference, and personnaly I prefer it this way, it's more clear what's happening to me.

Reply:
Yeah, it's just a js [styling convention](https://eslint.org/docs/rules/no-else-return), which we don't have in Eslint probably. I will let you decide if we want it, but I personally find it redundant.

Reply:
Yep I prefer it that way, so I won't touch it.